recipes/CommonVoice/ASR/CTC/hparams/train_de_with_wav2vec.yaml

```diff
@@ -0,0 +1,177 @@
+# Model: wav2vec2 + DNN + CTC
+# Augmentation: SpecAugment
+# Authors: Titouan Parcollet 2021
+#          Mirco Ravanelli 2021
+#          Sangeet Sagar 2022 (minor edits)
+# Language: German (DE)
+# ################################
+
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 8200
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+output_folder: !ref results/train_wav2vec2_char/<seed>
+wer_file: !ref <output_folder>/wer.txt
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+# Dataset prep parameters
+data_folder: !ref /ds/audio/CommonVoiceCorpus10.0/cv-corpus-10.0-2022-07-04/de
+train_tsv_file: !ref <data_folder>/train.tsv
+dev_tsv_file: !ref <data_folder>/dev.tsv
+test_tsv_file: !ref <data_folder>/test.tsv
+accented_letters: True
+language: de
+skip_prep: False
+
+# Data files
+train_csv: !ref <save_folder>/train.csv
+valid_csv: !ref <save_folder>/dev.csv
+test_csv: !ref <save_folder>/test.csv
+
+# We remove utterance slonger than 10s in the train/dev/test sets as
+# longer sentences certainly correspond to "open microphones".
+avoid_if_longer_than: 10.0
+
+# Training parameters
+number_of_epochs: 45
+lr: 1.0
+lr_wav2vec: 0.0001
+sorting: ascending
+auto_mix_prec: False
+sample_rate: 16000
+ckpt_interval_minutes: 30 # save checkpoint every N min
+
+# With data_parallel batch_size is split into N jobs
+# With DDP batch_size is multiplied by N jobs
+# Must be 6 per GPU to fit 16GB of VRAM
+batch_size: 8
+test_batch_size: 8
+num_workers:  8
+
+dataloader_options:
+   batch_size: !ref <batch_size>
+   num_workers: !ref <num_workers>
+test_dataloader_options:
+   batch_size: !ref <test_batch_size>
+   num_workers: !ref <num_workers>
+
+# BPE parameters
+# Be sure that the bos and eos index match with the BPEs ones
+blank_index: 0
+bos_index: 1
+eos_index: 2
+token_type: char  # ["unigram", "bpe", "char"]
+character_coverage: 1.0
+
+# Model parameters
+dnn_neurons: 1024
+wav2vec_output_dim: !ref <dnn_neurons>
+output_neurons: 32  # BPE size, index(blank/eos/bos) = 0
+
+# Functions and classes   
+epoch_counter: !new:speechbrain.utils.epoch_loop.EpochCounter
+   limit: !ref <number_of_epochs>
+
+augmentation: !new:speechbrain.lobes.augment.TimeDomainSpecAugment
+   sample_rate: !ref <sample_rate>
+   speeds: [95, 100, 105]
+
+enc: !new:speechbrain.nnet.containers.Sequential
+   input_shape: [null, null, !ref <wav2vec_output_dim>]
+   linear1: !name:speechbrain.nnet.linear.Linear
+      n_neurons: !ref <dnn_neurons>
+      bias: True
+   bn1: !name:speechbrain.nnet.normalization.BatchNorm1d
+   activation: !new:torch.nn.LeakyReLU
+   drop: !new:torch.nn.Dropout
+      p: 0.15
+   linear2: !name:speechbrain.nnet.linear.Linear
+      n_neurons: !ref <dnn_neurons>
+      bias: True
+   bn2: !name:speechbrain.nnet.normalization.BatchNorm1d
+   activation2: !new:torch.nn.LeakyReLU
+   drop2: !new:torch.nn.Dropout
+      p: 0.15
```

Comment:
same

Reply:
I initialize `dropout: 0.15` and then later use it as
```yaml
  drop: !new:torch.nn.Dropout
      p: !ref <dropout>
```